ZAPD/Main.cpp

```diff
@@ -37,22 +37,22 @@ void BuildAssetModelIntermediette(const std::string& outPath);
 void BuildAssetAnimationIntermediette(const std::string& animPath, const std::string& outPath);
 
 #if !defined(_MSC_VER) && !defined(__CYGWIN__)
-void ErrorHandler(int sig)
+void ErrorHandler(int32_t sig)
 {
 	void* array[4096];
-	const int nMaxFrames = sizeof(array) / sizeof(array[0]);
+	const int32_t nMaxFrames = sizeof(array) / sizeof(array[0]);
```

Comment:
This is a size variable, then it should use `size_t`, or at least an unsigned type. 
```suggestion
	const size_t nMaxFrames = sizeof(array) / sizeof(array[0]);
```